core/modules/qdisp/Executive.h

```diff
@@ -61,7 +63,7 @@ class QueryResource;
 class Executive : public std::enable_shared_from_this<Executive> {
 public:
     typedef std::shared_ptr<Executive> Ptr;
-    typedef std::map<int, std::shared_ptr<JobQuery>> JobMap;
+    typedef std::unordered_map<int, std::shared_ptr<JobQuery>> JobMap;
```

Comment:
It appears to take the about same amount of time at in2p3, but it should scale better. I would like to see how it performs on the pdac with 146,000 chunks. I'm not sure how I'm going to get a baseline on the pdac at this point.  It is, at least,  an easy change to make.